### ISSUE TITLE
Some Pokemon out of access due to non availability of RTC on some devices.

### ISSUE THREAD

**Issue by abhy-kumar** [Sierraffinity/CrystalDust#15]
Title. Some consoles and emulators do not feature a RTC chip which causes the players on them to be locked out of day/night pokemon. Is it possible to have a time toggle (like checking the clock in your room to reset it, etc.) to make these pokemon available to these players too?

**Comment by Sierraffinity** [comment]
Might consider making a "fake RTC" version available for download as a separate ROM, so people are aware they're explicitly opting into having to set the clock every time the game starts. I feel if I make it an option in the base game, people would abuse it or use it instead of just fixing their emulator. Not sure how I'm going to tackle this.

**Comment by huderlem** [comment]
An alternative to real RTC could be advancing time with VBlank ticks.

**Comment by abhy-kumar** [comment]
> Might consider making a "fake RTC" version available for download as a separate ROM, so people are aware they're explicitly opting into having to set the clock every time the game starts. I feel if I make it an option in the base game, people would abuse it or use it instead of just fixing their emulator. Not sure how I'm going to tackle this.

I do think this is the best approach for the reasons you mentioned.

**Comment by Sierraffinity** [comment]
> 
> 
> An alternative to real RTC could be advancing time with VBlank ticks.

Yeah, that would be what I do, my concern is more about how to present the option to players.
